util.js

```diff
@@ -56,6 +58,24 @@ function addPathRule(rules, rule) {
     }
 }
 
+function getBaseFeatureConfigs() {
+    const features = {};
+    // Grab all exception lists
+    const jsonListNames = fs.readdirSync(LISTS_DIR).filter((listName) => {
+        return listName !== UNPROTECTED_LIST_NAME && listName !== '_template.json';
+    });
```

Comment:
I'd also filter for `.json` suffix, just in case other hidden or cache files appear in that dir (e.g. when working with this locally).

Edit: I now see that this is just moving existing code, so that refactor suggestion is optional :)

Reply:
> Edit: I now see that this is just moving existing code, so that refactor suggestion is optional :)

Let me raise an issue for it, I'd rather not block here if you're ok with that. I can resolve tomorrow.

Reply:
Raised: https://app.asana.com/1/137249556945/project/1200890834746050/task/1210163485530445?focus=true